crates/sui-graphql-rpc/schema/draft_target_schema.graphql

```diff
@@ -595,7 +596,12 @@ type Epoch {
   fundOutflow: BigInt
 
   # TODO: Identify non-duplicate fields in `EndOfEpochInfo`
```

Comment:
We can also get rid of this TODO, I believe, because the fields in `EndOfEpochInfo` (on the last checkpoint in the epoch) should be indexed and used to update the relevant fields on the `epoch` table, when the epoch finishes.